psgeom/camera.py

```diff
@@ -351,7 +351,7 @@ def from_basisgrid(cls, bg, element_type=sensors.PixelArraySensor):
         
             pas = element_type(shape, 
                                pixel_shape, 
-                               type_name='grid_element_%dx%d' % shape, 
+                               type_name='MTRX:%d:%d:89:89' % shape, 
```

Comment:
we can't do this -- it hard codes the rayonix name to all cameras, and I need to re-use this code for all of them!

is it necessary, or just "nice"?